### DISCUSSION TITLE
project synopsis submission

### DISCUSSION THREAD

**Discussion by vyshnavi66** [github/india#621] [Students]
Where should we submit the project synopsis , inorder to activate project submission tab?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by Jitulteron7** [comment]
> Where should we submit the project synopsis , inorder to activate project submission tab?

Is your under review is complete? Mine is still under Review

**Reply by vyshnavi66** [reply]
> > Where should we submit the project synopsis , inorder to activate project submission tab?
> 
> Is your under review is complete? Mine is still under Review

I did not submit yet ! can I know your project name and what tools you are using?

**Reply by Jitulteron7** [reply]
sorry for the late reply. Well, I am a full stack developer (but more interest in backend ) I use nodejs for the backend so will apply to nodejs based projects

**Comment by hbindiresh** [comment]
> Where should we submit the project synopsis , inorder to activate project submission tab?

i think the date for a synopsis is changed to 12 of December

**Reply by vyshnavi66** [reply]
> > Where should we submit the project synopsis , inorder to activate project submission tab?
> 
> i think the date for a synopsis is changed to 12 of December

okay.
